helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -272,10 +273,38 @@ receivers:
           - move:
               from: log
               to: $$
+  {{- end }}
 
   {{- if .Values.logsCollection.extraFileLogs }}
   {{- toYaml .Values.logsCollection.extraFileLogs | nindent 2 }}
   {{- end }}
+
+  # https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/journaldreceiver
+  {{- if .Values.logsCollection.journald.enabled }}
+  {{- range $_, $unit := .Values.logsCollection.journald.units }}
+  {{- printf "journald/%s:" $unit.name | nindent 2 }}
+    directory: {{ $.Values.logsCollection.journald.directory }}
+    units: [{{ $unit.name }}]
+    priority: {{ $unit.priority }}
+    operators:
+    - type: metadata
+      resource:
+        com.splunk.source: {{ $.Values.logsCollection.journald.directory }}
+        com.splunk.sourcetype: 'EXPR("kube:"+$$._SYSTEMD_UNIT)'
```

Comment:
Is this the value that SCK uses for sourcetype? Now we have `kube:container:<conatiner_name>` and `kube:<journald_unit_name>`. Looks inconsistent to me. Shouldn't it be `kube:journald:<journald_unit_name>`?

cc @rockb1017

Reply:
Yeah, kube:journald: would make more sense. 

SCK has this format -  `kube:docker` for docker.service unit, `kube:kubelet` for kubelet.service unit